packages/core/src/amazonq/lsp/types.ts

```diff
@@ -88,12 +88,44 @@ export const GetIndexSequenceNumberRequestType: RequestType<GetRepomapIndexJSONR
     'lsp/getIndexSequenceNumber'
 )
 
-export type ContextCommandItemType = 'file' | 'folder'
+export type ContextCommandItemType = 'file' | 'folder' | 'code'
+
+export type SymbolType =
+    | 'Class'
+    | 'Function'
+    | 'Interface'
+    | 'Type'
+    | 'Enum'
+    | 'Struct'
+    | 'Delegate'
+    | 'Namespace'
+    | 'Object'
+    | 'Module'
+    | 'Method'
+
+export interface Position {
+    line: number
+    column: number
+}
+export interface Span {
+    start: Position
+    end: Position
+}
+
+// LSP definition of DocumentSymbol
+
+export interface DocumentSymbol {
```

Comment:
Can we reuse this type from `vscode-languageserver-node` or `vscode-languageserver-types`?

Reply:
Unfortunately no.